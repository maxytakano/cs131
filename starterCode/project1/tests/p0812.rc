function : void foo() {
	bool x = 2;
	float y = 3;
	int z = 4.0;
}