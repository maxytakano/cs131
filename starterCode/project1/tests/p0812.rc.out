Error, "/tmp/cs131_testrunner_ZxTaux/file.rc", line 2: 
  Initialization value of type int not assignable to constant/variable of type bool.
Error, "/tmp/cs131_testrunner_ZxTaux/file.rc", line 4: 
  Initialization value of type float not assignable to constant/variable of type int.
Compile: failure.
